internal/service/ec2/validate_test.go

```diff
@@ -105,3 +106,46 @@ func TestValid4ByteASN(t *testing.T) {
 		}
 	}
 }
+
+func TestValidateIPv4OrIPv6(t *testing.T) {
+	validator := validateIPv4OrIPv6(
+		validation.IsCIDRNetwork(16, 24),
+		validation.IsCIDRNetwork(40, 64),
+	)
+	validCIDRs := []string{
```

Comment:
Its probably a good idea to include a comment on why these are invalid and include a link to documentation, same with #L131

Reply:
Done